app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.dwarves import dwarf
+from app.players.elves import elf
+
+
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
better to clarify the type of instances inside of list

Reply:
still not fixed